packages/core/src/js/feedback/FeedbackWidget.tsx

```diff
@@ -33,6 +33,7 @@ export class FeedbackWidget extends React.Component<FeedbackWidgetProps, Feedbac
     ...defaultConfiguration
   }
 
+  private static _didSubmitForm: boolean = false;
```

Comment:
`_didSubmitForm` should not be static since it's related to the form instance.

Reply:
Good catch @krystofwoldrich 🙇 
Updated with f17106e502694606c1428201f0b568706dddf15d